src/components/SRNEntry.vue

```diff
@@ -105,56 +105,71 @@ export default {
     return {
       userIDList: [{ userID: "", valid: false }], //array containing user-id's and a valid flag for each
       invalidInputMessage: null, //flag to check if the input is in correct format
-      isUserValid: false, // whether the user exists in the backend database
+      isCurrentUserValid: false, // temp flag which contains the validity of the most recent user ID
       maxLengthOfSRN: 10,
       validateCount: 0, //this variable tells us how many times the user has been validated.
       invalidLoginMessage: "Please enter correct SRN / कृपया सही SRN दर्ज करें",
-      isLoading: false,
+      isLoading: false, 
```

Comment:
I think your linter is not working maybe?
Because in my and Aman's system, whenever we save a `.vue` file, it automatically lints everything.
So this extra space that has been added, should've been removed automatically

Can you check once if your linter is working fine?